samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -92,14 +91,37 @@ static int remove_whitespace(char *buf)
 	return 0;
 }
 
+/* remove trailing AT OK message */
+static void remove_ok(char *buf)
+{
+	uint8_t *ok_resp;
+	uint32_t index;
+
+	remove_whitespace(buf);
```

Comment:
`remove_whitespace` is already called at the end of `query_modem`.

Reply:
`remove_whitespace` can be called multiple times on the same buffer without affecting it if there are no whitespaces. Since I wanted to make `remove_ok` like a generically useful function I added the call since if there is whitespace it will get removed, if not it will do nothing. However, upon closer inspection, I've been thinking that making a function just to remove the trailing "OK" is a bit useless so I moved the code where it's needed. I would argue that `remove_whitespace` can be managed in the same way since it's only used once in `query_mode`, but I would prefer to keep the changes in this PR minimal and related to the topic.